site/app/templates/plagiarism/PlagiarismConfigurationForm.twig

```diff
@@ -22,167 +23,217 @@
+                                                    selected
+                                                {% endif %}
+                                        >
+                                            {{ sem }}
+                                        </option>
+                                    {% endfor %}
+                                </select>
+                                <select name="prev_course_{{ loop.index0 }}">
+                                    <option value="">None</option>
+                                    {% for sem_course, course_gradeables in prior_term_gradeables[saved_prev_sem] %}
+                                        <option value="{{ sem_course }}"
+                                                {% if sem_course == saved_prev_course %}
+                                                    selected
+                                                {% endif %}
+                                        >
+                                            {{ sem_course }}
+                                        </option>
+                                    {% endfor %}
+                                </select>
+                                <select name="prev_gradeable_{{ loop.index0 }}">
+                                    <option value="">None</option>
+                                    {% for course_gradeable in prior_term_gradeables[saved_prev_sem][saved_prev_course] %}
+                                        <option value="{{ course_gradeable }}"
+                                                {% if course_gradeable == saved_prev_gradeable %}
+                                                    selected
+                                                {% endif %}
+                                        >
+                                            {{ course_gradeable }}
+                                        </option>
+                                    {% endfor %}
+                                </select>
+                                <br />
+                            {% endfor %}
+                        {% endif %} #}
+                        {#{% set saved_prev_term_grad_length = saved_config['prev_term_gradeables']|length %}
+                        <select name="prev_sem_{{ saved_prev_term_grad_length }}">
+                            <option value="">None</option>
+                            {% for sem, sem_courses in prior_term_gradeables %}
+                                <option value="{{ sem }}">{{ sem }}</option>
+                            {% endfor %}
+                        </select>
+
+                        <select name="prev_course_{{ saved_prev_term_grad_length }}">
+                            <option value="">None</option>
+                        </select>
+                        <select name="prev_gradeable_{{ saved_prev_term_grad_length }}">
+                            <option value="">None</option>
+                        </select>#}
```

Comment:
Why is this commented out? Is it just temporary and will be re-added in a near-term future PR?

Reply:
Yes.  There is currently no complete, or nearly-complete backend for this, either in the Lichen codebase or in the PHP so we just commented this out for the moment instead of trying to get something together which only partially worked.  We'll hopefully be making PRs to implement these features soon.